.gitignore

```diff
@@ -1,5 +1,24 @@
+# Composer Dependencies
 /vendor/*
+/bin
```

Comment:
Near as I can tell (and really I mean guess at) is that they only exist in the Cake App Project.

So I guess that means we need them in our project for new 3.0 projects, but since we drop other bin files in there then yeah, we want to ignore everything but the cake collection of bin scripts: http://book.cakephp.org/3.0/en/appendices/3-0-migration-guide.html#console